form.yml

```diff
@@ -6,6 +6,51 @@ hana:
     $type: boolean
     $default: true
     $help: Install all required packages from currently existing repositories
+    $optional: true
+  saptune_solution:
```

Comment:
This should have `HANA` as default, right?

Reply:
Thanks @arbulu89 yes I will update this.